src/ViewHelpers/PaginationViewHelper.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+
+namespace Portal\ViewHelpers;
+
+class PaginationViewHelper
+{
+
+    /**
+     * Takes in count of pages required and current page, and returns pagination for applicants list.
+     *
+     * @param $page
+     * @param $count
+     *
+     * @return string
+     */
+    public static function applicantsPagination($page, $count)
+    {
+        $next = (strval($page+1));
+        $prev = (strval($page-1));
+
+        $applicantPagination = '';
+
+        $applicantPagination .= '<nav aria-label="Page navigation mt-5">';
+        $applicantPagination .= '<ul class="pagination justify-content-center">';
+        $applicantPagination .= '<li class="page-item ';
+        if ($page <= 1) {
+            $applicantPagination .= 'disabled';
+        }
+        $applicantPagination .= '"><a class="page-link" href="';
+        if ($page <= 1) {
+            $applicantPagination .= '#';
+        } else {
+            $applicantPagination .= '/applicants?page=' . $prev;
```

Comment:
If you remove '/applicants' from each of these links, it should still work and would mean that this view helper could be used universally across the app whenever we want some pagination

Reply:
Then you could change the method name so it's more generic